### PR TITLE
use git-checkout libucontext

### PR DESCRIPTION
**Pull request by mesaglio**

### CODE REVIEW

**Comment by mesaglio** [comment]
@hectorj2f

File: libucontext.yaml [hunk 1]
```diff
@@ -44,10 +45,15 @@ subpackages:
...
+      - runs: |
+          make docs install_docs DESTDIR="${{targets.contextdir}}"
       - uses: split/manpages
     description: libucontext manpages
 
 update:
   enabled: true
-  release-monitor:
-    identifier: 17085
+  github:
```
Comment (hectorj2f) [inline libucontext.yaml:55]: Could we also add a test here?

**Review by hectorj2f — COMMENTED** [review]
LGTM

File: libucontext.yaml (hunk 1 above)
Comment (mesaglio) [reply libucontext.yaml:55]: a test to validate if `libucontext.so` is in the apk?

**Review by hectorj2f — APPROVED** [review]

**Review by ajayk — APPROVED** [review]

**Pull request merged**